location_service/insert_database/insert.py

```diff
@@ -0,0 +1,44 @@
+from sqlalchemy import create_engine, text
+from dotenv import load_dotenv
+import pandas as pd
+import os
+
+load_dotenv()
+DB_HOST = os.getenv("DB_HOST")
+DB_NAME = os.getenv("DB_NAME")
+DB_USER = os.getenv("DB_USER")
+DB_PASSWORD = os.getenv("DB_PASSWORD")
+
+engine = create_engine(f"mysql+pymysql://{DB_USER}:{DB_PASSWORD}@{DB_HOST}/{DB_NAME}")
+
+df = pd.read_csv("location_service/insert_database/sample_coordinates.csv")
+
+# INSERT_MODE can be 'fail', 'replace', or 'append'
+INSERT_MODE = "replace"
+
+with engine.connect() as connection:
+    try:
+        if INSERT_MODE == "fail":
```

Comment:
What's the point of `fail` mode? We should have a workflow that looks like this:

- User adds data into the db
- If there's data already in the db, throw an error (and don't change the database)

- User now adds the data to the db and can decide if they want to **overwrite** the table or **append** to it (although most times it'll be overwriting)

Reply:
You can make a function like:
```
def function(..., mode=None):
  if mode == None:
    if database_table_populated"
      throw Error
  if mode == "overwrite:
    delete database_table
    insert data
  if mode == "append":
    insert data
```